packages/terra-navigation-prompt/CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ Unreleased
 ------------------
 ### Changed
 * Missing aria-labels added in examples.
+* Changed 'NavigationPromptCheckpoint' to have the same logic as 'notification-dialog' changes
```

Comment:
This needs to be moved to under the unreleased section

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/899/commits/3aac8e22f188f85fb1054f1de910709b7f71e752